### ISSUE TITLE
Cloud Connector Configuration fails

### ISSUE THREAD

**Issue by LiohMoeller** [theforeman/foreman_rh_cloud#814]
I am trying to Configure the Cloud Connector using **Configure** / **Inventory Upload** / **Configure Cloud Connector** which invokes an Ansible Job run. This fails due to a missing role 'redhat.satellite_operations.cloud_connector'.
```
ERROR! the role 'redhat.satellite_operations.cloud_connector' was not found in /tmp/d20230323-802194-1r0ilbc/project/roles:/etc/ansible/roles:/usr/share/ansible/roles:/tmp/d20230323-802194-1r0ilbc/project
The error appears to be in '/tmp/d20230323-802194-1r0ilbc/project/playbook.yml': line 6, column 7, but may
be elsewhere in the file depending on the exact syntax problem.
The offending line appears to be:
  roles:
      - redhat.satellite_operations.cloud_connector
      ^ here
Exit status: 1
```

**Comment by LiohMoeller** [comment]
It seems the missing role is part of https://github.com/RedHatSatellite/satellite-operations-collection 

But as this is Foreman, shouldn't it be https://github.com/theforeman/foreman-operations-collection here?

The package ansible-collection-theforeman-operations is installed on the Foreman Server.

**Comment by LiohMoeller** [comment]
Besides that, _cloud_connector/tasks/main.yml_ tries to install yggdrasil-worker-forwarder which does not seem to be available in the Foreman repositories.

**Comment by evgeni** [comment]
You should be able to replace `redhat.satellite_operations` with `theforeman.operations` just fine.

As for the forwarder package, that's a good question!

**Comment by LiohMoeller** [comment]
Yes, I have created a PR for that: https://github.com/theforeman/foreman_rh_cloud/pull/815

The yggdrasil-worker-forwarder seems only to be available in downstream Satellite Repositories. Did not find the relevant upstream .spec as well.

**Comment by LiohMoeller** [comment]
Upstream Project itself resides here: https://github.com/shimshtein/yggdrasil-worker-forwarder - Pipeline does not produce RPMs. Spec file not available. The resulting package should be part of the Foreman Repository as well.

**Comment by evgeni** [comment]
there was a typo in our repo config, just fixed it and the package should show up in the next ~5 min

edit: published for 3.4, 3.5, 3.6 and nightly
